### ISSUE TITLE
Y24-283 - Update Yard documentation to include proper descriptions for resources

### ISSUE THREAD

**Issue by dasunpubudumal** [sanger/sequencescape#4302]
**User story**
As developers, we need to update Yard documentation hosted at https://sanger.github.io/sequencescape/ to include proper overview sections for resources in [`Api::V2`](https://sanger.github.io/sequencescape/Api/V2.html)

**Who are the primary contacts for this story**
Dasun, Steve, Katy, Abdullah, Stephen H

**Who is the nominated tester for UAT**

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Add a detailed description on what each resource represents. This needs to include the business value of that resource.
- [ ] Add any example usage of API or cURL

**Dependencies**
This story is blocked by the following dependencies:

References
This story has a non-blocking relationship with:

**Additional context**

**Comment by dasunpubudumal** [comment]
Closing this in favour of https://github.com/sanger/sequencescape/issues/4522 (duplicate).
